pages/about.html

```diff
@@ -1,12 +1,33 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link href="style.css" rel="stylesheet">
 </head>
 <body>
-  
-</body>
+    <div class="grid-container">
+        <div class="item1"><h1>Jessica Edith Ambriz-Madrigal </h1></div>
+        <div class="item2">
+        <div><p><a href="file:///Users/jessicaambriz-madrigal/Developer/projects/personal-portfolio-site/pages/index.html"> Home </a></p></div>
```

Comment:
Be careful with your links! This is looking for a specific file on your computer. These links do not work if other people are trying to open your webpage. Instead, you should use relative paths.
Here is a great resource on absolute vs relative paths:  https://www.geeksforgeeks.org/absolute-relative-pathnames-unix/

This applies to image URLs as well.